server/manager/src/main/java/org/apache/accumulo/manager/TabletGroupWatcher.java

```diff
@@ -783,13 +784,24 @@ private void mergeMetadataRecords(MergeInfo info) throws AccumuloException {
       TabletColumnFamily.PREV_ROW_COLUMN.fetch(scanner);
       ServerColumnFamily.TIME_COLUMN.fetch(scanner);
       ServerColumnFamily.DIRECTORY_COLUMN.fetch(scanner);
+
       scanner.fetchColumnFamily(DataFileColumnFamily.NAME);
+      scanner.fetchColumnFamily(CurrentLocationColumnFamily.NAME);
+      scanner.fetchColumnFamily(LogColumnFamily.NAME);
       Mutation m = new Mutation(stopRow);
       MetadataTime maxLogicalTime = null;
       for (Entry<Key,Value> entry : scanner) {
         Key key = entry.getKey();
         Value value = entry.getValue();
-        if (key.getColumnFamily().equals(DataFileColumnFamily.NAME)) {
+
+        // Verify that Tablet is offline
+        if (key.getColumnFamily().equals(CurrentLocationColumnFamily.NAME)) {
```

Comment:
Can also check for a future location, should not see that either.

```suggestion
        if (key.getColumnFamily().equals(CurrentLocationColumnFamily.NAME)  || key.getColumnFamily().equals(FutureLocationColumnFamily.NAME)) {
```